modules/middleware/src/flattenAggregations.js

```diff
@@ -18,9 +18,13 @@ function flattenAggregations({ aggregations, includeMissing = true }) {
         ...value.buckets,
         ...(includeMissing && missing ? [{ ...missing, key: MISSING }] : []),
       ];
+      const bucket_count = buckets.filter(bucket => bucket.doc_count > 0)
+        .length;
```

Comment:
do we really need this filter? other than `MISSING`, are there any other expected case of `doc_count <= 0` that can show up?

If we can avoid it that'd be best to reduce load on the main thread